test/metabase/driver/presto_test.clj

```diff
@@ -123,12 +123,6 @@
         (update :category_id int)
         (update :id int))))
 
-;;; FIELD-PERCENT-URLS
-(datasets/expect-with-engine :presto
-  0.5
-  (data/dataset half-valid-urls
-    (sql/field-percent-urls (PrestoDriver.) (db/select-one 'Field :id (data/id :urls :url)))))
-
```

Comment:
we are switching from checking for a specific threshold and marking something as a url to keeping track of the fraction of entries that are urls, putting that in a fingerprint and using that to classify things later.